### PR TITLE
Minor updates to README.

### PR DESCRIPTION
**Pull request by FintanK**
Just so devs know where to find the plugins when finding parched in a project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,13 +10,15 @@ It is heavily inspired by Brunch.
 
 See [parched-example-app](https://github.com/raisedmedia/parched-example-app), or [parched-tasks-webapp](https://github.com/raisedmedia/parched-tasks-webapp) even. The following will be based on them.
 
+A list of ready made plugins can be found at [parched-plugins](https://github.com/BraveNewWorldDev/parched-plugins)
```
Comment (mikew) [inline README.md]: This would be a better link for that: https://www.npmjs.com/browse/keyword/parched

**Comment by FintanK** [comment]
Ah I didn't realized they were published. Great.

File: README.md [hunk 2]
```diff
@@ -10,13 +10,15 @@ It is heavily inspired by Brunch.
...
+
 ## Getting Started
 
 ```bash
 npm install --save gulp parched
 ```
 
-Then update your `Gulpfile.js`
+Then update your `Gulpfile.js`. You can specify an plugins you wish to load here.
```
Comment (mikew) [inline README.md]: Plugins are specified as dependencies in your `package.json`, not passed to `Parched.setup`

Comment (FintanK) [inline README.md]: Oh I see, so they're picked up automatically? That's handy.

**Comment by FintanK** [comment]
Squashed the commits for cleaner VC changes.

File: README.md (hunk 2 above)
Comment (FintanK) [inline README.md]: Removed comment and squashed commits.

File: README.md [hunk 3]
```diff
@@ -10,13 +10,15 @@ It is heavily inspired by Brunch.
 
 See [parched-example-app](https://github.com/raisedmedia/parched-example-app), or [parched-tasks-webapp](https://github.com/raisedmedia/parched-tasks-webapp) even. The following will be based on them.
 
+A list of ready made plugins can be found here [parched-plugins](https://www.npmjs.com/browse/keyword/parched)
```
Comment (mikew) [inline README.md]: Super nit-picky, I know, but could you change the copy to read: `A list of ready-made plugins [can be found on npm](https://www.npmjs.com/browse/keyword/parched).`

Comment (FintanK) [inline README.md]: updated!

**Pull request merged**